src/__tests__/command.spec.ts

```diff
@@ -219,24 +219,51 @@ describe('buildkite-graph', () => {
                 ),
             ]);
 
-            createTest('plugins', () =>
-                new Pipeline('whatever').add(
-                    new CommandStep('noop').plugins
-                        .add(
-                            new Plugin('bugcrowd/test-summary#v1.5.0', {
-                                inputs: [
-                                    {
-                                        label: ':htmllint: HTML lint',
-                                        artifact_path:
-                                            'web/target/htmllint-*.txt',
-                                        type: 'oneline',
-                                    },
-                                ],
-                            }),
-                        )
-                        .plugins.add(new Plugin('detect-clowns#v1.0.0')),
-                ),
-            );
+            describe('plugins', () => {
+                const plugins = [
+                    new Plugin('bugcrowd/test-summary#v1.5.0', {
+                        inputs: [
+                            {
+                                label: ':htmllint: HTML lint',
+                                artifact_path: 'web/target/htmllint-*.txt',
+                                type: 'oneline',
+                            },
+                        ],
+                    }),
+                    new Plugin('detect-clowns#v1.0.0'),
+                ];
+                const stepWithPlugins = new CommandStep('noop').plugins
+                    .add(plugins[0])
+                    .plugins.add(plugins[1]);
```

Comment:
This needs to go into a beforeEach to ensure we're not leaking any state between tests. Alternatively (and I think that might be easier), you can just make up a second step with plugins for the second test on the fly.

Reply:
Done in 6a858bb. Used `beforeEach()`.